specification/cognitiveservices/data-plane/QnAMaker/readme.md

```diff
@@ -22,7 +22,6 @@ input-file: stable/v4.0/QnAMaker.json
 ```
 
 ``` yaml
-
 tag: runtime_release_4_0
```

Comment:
@rokulka Have you tested with `autorest`? It doesn't look correct.

 As you have `tag: runtime_release_4_0` and  `tag: release_4_0` under `batch:`, the tags in here should be removed.

Please try `autorest` against the configuration and test the auto-generated code.

Reply:
We did try and it worked. Here is the o/p
  Loading AutoRest extension '@microsoft.azure/autorest.csharp' (~2.3.79->2.3.82)
   Loading AutoRest extension '@microsoft.azure/autorest.modeler' (2.3.55->2.3.55)
Processing batch task - {"tag":"release_4_0"} .
Processing batch task - {"tag":"runtime_release_4_0"} .

Reply:
My mistake, I was using an edited version. Sorry for the inconvenience. 
Seems to be all set!